src/elements/violin.js

```diff
@@ -32,7 +33,8 @@ const Violin = Chart.elements.Violin = ArrayElementBase.extend({
 
 		ctx.beginPath();
 		if (vert) {
-			const {x, width} = vm;
+			const x = vm.x;
```

Comment:
shouldn't be padding on both sides, i.e. the box centered => also the x value has to change, hasn't it?